BrAddressForm.vue

```diff
@@ -6,6 +6,7 @@
           v-model="streetAddress.value"
           :float-label="streetAddress.label"
           class="q-pa-sm q-mt-md"
+          autocomplete="street-address"
```

Comment:
Given that this only takes a single line as input, it seems like perhaps this isn't the right autocomplete value?

https://www.w3.org/TR/html52/sec-forms.html#inappropriate-for-the-control

Perhaps this is handled via some translation when applied to a field that doesn't accept new lines? The spec seems to say that `street-address` is for multiline controls only... despite it appearing in various online examples using `input`.

Reply:
[according to mozzilla street address **can be** multi-line, but is not required to be.](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#Administrative_levels_in_addresses)

[I actually think after reviewing global addresses](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country) we might consider changing street-address to a text field that is multi-line.

for instance the U.S. only uses address-level1 and 2, however China and Belgium uses all 4 levels which means someone might want to list their:

Region, Province, District and Municipalities
this might actually be a good idea with the current credentials too.
the more location data we can make verifiable the more valuable that information is.

We support Region and Province right now, but if I wanted to add District and Municipality I would have to put both on the same line.

Reply:
this for instance is an address of a co-working space in Beijing:

中國北京市朝阳区大望路和乔大厦 邮政编码: 100025, Chaoyang District, Beijing, China

Country and Region we cover but the rest of that the Destrict and the building info would have to fit on a single line. Also the one line input does not cover Condos and Apartments where the address might include a building number, floor, apartment number, and other information.

Reply:
@dlongley I think we need to build a list of address we want to support.
So for instance we cover everything from an apartment in Dhaka to a household in Montana.

Reply:
- [ ] change q-input type to textarea

Reply:
@dlongley can you recheck and comment on this? I changed it to a textarea seems like the best solution.

Reply:
We definitely don't want a textarea here, it will result in more user error; we want an input field. The HTML5 spec tells us not to use `street-address`, so let's go with `address-line1`. We should make `address-line2` optional. See this link for a lot of user feedback on address forms:

https://baymard.com/blog/address-line-2

Reply:
ok cool will do that in a moment right now changing the build process for countries.js so that is pre-formated for the Select.

Reply:
The article you linked to actually suggests that address-line 2 will cause users to come to a halt and comments below the article suggest that a textarea with a limit on the length is fine and in fact might be better for mobile users.

> Christian, Baymard Institute NOVEMBER 11, 2014
> 
> Hi Tomas, yes as mentioned in point #5 in the article on can also consider a multi-line address field. We don’t have any particular test observations on this implementation, but the size of the field will allow it to display most inputs in full (without partly masking the data – as Laura also mention as an issue with a single line address field).

is there more data somewhere on the textarea?

Reply:
@dlongley this feature did not make this release however I am adding it to the doc. we do need more info though on textarea. as it stands right now both of the articles I found recommended a hidden second address-line2 so will try it in next PR.

Reply:
Making this a regular input with autocomplete set to address-line1. Will make an issue to add in address-line2 support in a future release.

Reply:
Addressed here: [diff](https://github.com/digitalbazaar/bedrock-vue-address-form/compare/9a72c4e8169ac2227107f256fdfe50153d1107d2..df5f4106f947ad984144a311d9313dceb79deeeb)